DataCollection/src/youtubecollector/transcripts.py

```diff
@@ -0,0 +1,61 @@
+import os
+import glob
+import csv
+import webvtt
+from os.path import isfile
+
+
+def get_transcripts(vtt_folder):
+    """:param vtt_folder should be location string ending in *.vtt to get all .vtt files like "files/output/*.vtt" """
+
+    video_ids = []
+    transcripts = []
+
+    for filename in glob.glob(vtt_folder):
+        ids = get_ids_from_filename(filename)
+        video_ids.append(ids)
+
+        try:
+            words = []
+            for caption in webvtt.read(filename):
+                words.append(caption.text)
+            transcripts.append(words)
+        except:
+            pass
+
+    with open('captions.csv', 'w') as csv_file:
+        header = ['videoId', 'transcript']
+        writer = csv.writer(csv_file, delimiter=',', fieldnames=header)
+
+        if not isfile('captions.csv'):
+            writer.writeheader()
+        writer.writerows(zip(video_ids, transcripts))
+
+
+def get_ids_from_filename(filename):
```

Comment:
Will make this "private" function
```suggestion
def _get_ids_from_filename(filename):
```